bazel/envoy_select.bzl

```diff
@@ -67,6 +67,16 @@ def envoy_select_wasm_v8(xs):
         "//conditions:default": xs,
     })
 
+# Selects True or False depending on the Wasm runtimes enabled in the current build.
+def envoy_select_wasm_v8_bool():
+    return select({
+        "@envoy//bazel:wasm_wamr": False,
+        "@envoy//bazel:wasm_wasmtime": False,
+        "@envoy//bazel:wasm_wavm": False,
+        "@envoy//bazel:wasm_none": False,
+        "//conditions:default": True,
```

Comment:
Nit: should v8 be an explicit target here, rather than implicit in default?

Reply:
But it's both an explicit target (`@envoy//bazel:wasm_v8` when built with `--define wasm=v8` flag) and the implicit default (when buith without any `--define wasm=xxx` flag). `//conditions:default` covers both.

I've looked at `selects`, but I don't see a way to make `@envoy//bazel:wasm_v8` also cover the default case without `match_none` feature (that doesn't exist). Any hints?

Reply:
Does it fail when you explicitly list `@envoy//bazel:wasm_v8` as a condition? 

I think maybe just add a comment explaining the fact that the deafult covers this target is fine in any case.

Reply:
1. I've added `@envoy//bazel:wasm_v8`, but it doesn't cover the default case, so tests without `--define wasm=v8` fail.
2. There has to be `//conditions:default`, otherwise not all conditions are covered and Bazel complains.
3. I've added comment to the default case.

PTAL.